src/components/companion/blueprintProfiles/additionalLinks.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+import { AnimatedBorder } from '../../ui/animated-border';
+
+interface UserProfile {
+    additionalLinks: string[];
+}
+
+interface AdditionalLinksProps {
+    userData: UserProfile;
+}
+
+const AdditionalLinks: FC<AdditionalLinksProps> = ({ userData }) => {
+    return (
+        <AnimatedBorder className="w-full mb-3 sm:mb-4">
+        <div className="rounded-lg p-3 sm:p-4">
+            <span className="text-xs sm:text-sm text-light-grey font-mono mb-1 sm:mb-2">ADDITIONAL LINKS</span>
+            {userData.additionalLinks.map((link: string, index: number) => (
```

Comment:
Note that we've only provided the option for **one** additional link on the registration form, so this will become a single string, not an array. You may need to update the UserData interface too